ArcGISRuntimeSDKQt_SampleViewers/ArcGISRuntimeSDKQt_Samples/SampleManager.h

```diff
@@ -52,6 +52,8 @@ class SampleManager : public QObject
   Q_PROPERTY(bool downloadInProgress READ downloadInProgress WRITE setDownloadInProgress NOTIFY downloadInProgressChanged)
   Q_PROPERTY(QString downloadText READ downloadText WRITE setDownloadText NOTIFY downloadTextChanged)
   Q_PROPERTY(double downloadProgress READ downloadProgress WRITE setDownloadProgress NOTIFY downloadProgressChanged)
+  Q_PROPERTY(bool cancelDownload READ cancelDownload WRITE setCancelDownload NOTIFY cancelDownloadChanged)
```

Comment:
should it be `downloadCanceled`? `cancelDownload` sounds more like an action or method

Reply:
When downloading all offline data, after each download finishes, we check to see if `cancelDownload` is true or not before proceeding to download the next item in the downloadQueue. I opted to word is that way because it's like a flag to take an action in the future, whereas `downloadCanceled` implies to me that something happened already.